scalafix-tests/unit/src/test/scala/scalafix/tests/SyntacticTests.scala

```diff
@@ -9,24 +9,31 @@ import java.nio.file.Files
 import java.nio.file.Paths
 import scalafix.internal.rule.ProcedureSyntax
 
-class ErrorSuite extends SyntacticRuleSuite(ProcedureSyntax) {
+class ErrorSuite extends SyntacticRuleSuite {
   test("on parse error") {
     intercept[ParseException] {
       ProcedureSyntax.apply(Input.String("object A {"))
     }
   }
 }
-class PatchSuite
-    extends SyntacticRuleSuite(Rule.syntactic("PatchSuite")(ctx =>
-      ctx.addRight(ctx.tree.tokens.find(_.is[Ident]).get, "bba"))) {
+class PatchSuite extends SyntacticRuleSuite {
 
-  val original =
+  val addRightRule: Rule = Rule.syntactic("addRight") { (ctx) =>
+    ctx.addRight(ctx.tree.tokens.find(_.is[Ident]).get, "bba")
+  }
+
+  val addLeftRule: Rule = Rule.syntactic("addLeft") { (ctx) =>
```

Comment:
Can we move this definition right next to the next where it's used?

Reply:
Done.